### PR TITLE
Doc: Correct fscache distribution steps

### PR DESCRIPTION
**Pull request by yqleng1987**
update compile command and copy the binary into $PATH in docs/nydus-fscache.md

### CODE REVIEW

**Review by jiangliu — APPROVED** [review]

File: docs/nydus-fscache.md [hunk 1]
```diff
@@ -37,10 +37,17 @@ CONFIG_EROFS_FS_ONDEMAND=y
...
 ``git clone https://github.com/dragonflyoss/image-service.git``
 
-3. Build nydusd with \
-``cargo build --target x86_64-unknown-linux-gnu --features=fusedev --release --target-dir target-fusedev --bin nydusd``
+3. Build nydusd and nydus-image with 
 
-4. Build ctr-remote with
+``` bash
+cd image-service
+make
```
Comment (imeoer) [inline docs/nydus-fscache.md:44]: Should use `make release`?

Comment (yqleng1987) [reply docs/nydus-fscache.md]: fixed

**Review by hsiangkao — APPROVED** [review]

File: docs/nydus-fscache.md [hunk 2]
```diff
@@ -37,10 +37,17 @@ CONFIG_EROFS_FS_ONDEMAND=y
...
+3. Build nydusd and nydus-image with 
 
-4. Build ctr-remote with
+``` bash
+cd image-service
+make release
+```
+
+4. Copy the "nydus-image" binary file compiled in Step 3 into _$PATH_ e.g. /usr/local/bin with \
+``cp target-fusedev/release/nydus-image /usr/local/bin``
```
Comment (imeoer) [inline docs/nydus-fscache.md:48]: Should be `cp target-fusedev/x86_64-unknown-linux-gnu/release/nydus-image /usr/local/bin` ?

Comment (hsiangkao) [reply docs/nydus-fscache.md:48]: `x86_64-unknown-linux-gnu` isn't occured on my side as well.

**Pull request merged**